manifests/init.pp

```diff
@@ -72,7 +73,8 @@
   }
 
   file { $dhcp_dir:
-    mode    => '0755',
+    group   => $dhcp_root_group,
```

Comment:
I wonder if we should also manage the owner.

Reply:
Of the config directory? I think it's out of scope of this PR.

Reply:
Why is the group in scope but the owner isn't?

Reply:
we are just adding the existing group to the directory as well here. But if you think we should add a user configuration as well... :) We should just add it to the all config files as well and it rises question if we want to separate those configurations (dir/file) or not :)

Reply:
Let me add it as a separate commit and let's see :+1: